src/main/java/teammates/common/util/StringHelper.java

```diff
@@ -181,7 +183,8 @@ public static String decrypt(String message) throws InvalidParametersException {
     }
 
     public static String toDecimalFormatString(double doubleVal) {
```

Comment:
Let's generalize the method so we can reuse it everywhere (at the moment the method doesn't make sense anyway).

And let's use `NumberFormat.getNumberInstance` instead since that's the way Java designed it and also [documented in the Java tutorials](https://docs.oracle.com/javase/tutorial/i18n/format/decimalFormat.html).

 `To obtain standard formats for a given locale, use the factory methods on NumberFormat such as getInstance or getCurrencyInstance. If you need only minor adjustments to a standard format, you can modify the format returned by a NumberFormat factory method.`

https://docs.oracle.com/javase/8/docs/api/java/text/DecimalFormat.html#DecimalFormat-java.lang.String-java.text.DecimalFormatSymbols-

Reply:
> Let's generalize the method so we can reuse it everywhere

How exactly? I can use this function as it is already in other places